plugins/by-name/codeium-nvim/default.nix

```
@@ -15,108 +21,38 @@ lib.nixvim.plugins.mkNeovimPlugin {
     khaneliman
   ];
 
-  # TODO: added 2024-09-03 remove after 24.11
-  deprecateExtraOptions = true;
-  optionsRenamedToSettings = [
-    "configPath"
-    "binPath"
-    [
-      "api"
-      "host"
-    ]
-    [
-      "api"
-      "port"
-    ]
-    [
-      "tools"
-      "uname"
-    ]
-    [
-      "tools"
-      "uuidgen"
-    ]
-    [
-      "tools"
-      "curl"
-    ]
-    [
-      "tools"
-      "gzip"
-    ]
-    [
-      "tools"
-      "languageServer"
-    ]
-    "wrapper"
-  ];
+  description = ''
+    By default, Nixvim will install the `curl`, `gzip`, `coreutils`, `util-linux` and `codeium` packages and populate settings.tools.*` accordingly.
+
+    You are free to override `dependencies.*.enable` and `dependencies.*.package` to respectively disable and customize this behavior.
```

Wording suggestion, if you agree:
```suggestion
    You are free to configure `dependencies.*.enable` and `dependencies.*.package` to disable or customize this behavior, respectively.
```